frontend/app/views/spree/products/index.html.erb

```diff
@@ -1,7 +1,9 @@
 <% content_for :sidebar do %>
   <div data-hook="homepage_sidebar_navigation">
     <% if "spree/products" == params[:controller] && @taxon %>
-      <%= render :partial => 'spree/shared/filters' %>
+      <% Spree::Config.search_filter_presenter_class.new(taxon: @taxon).tap do |presenter| %>
```

Comment:
Yes, I agree and I don't know what's better. Does anyone else have an opinion on this? 
Please, feel free to point us to any issue on this topic or a place in the Solidus codebase where we adopted a similar approach?

cc/ @cbrunsdon @jhawthorn